models/staging/edfi_3/stage/stg_ef3__parents.sql

```diff
@@ -0,0 +1,26 @@
+with base_parents as (
+    select * from {{ ref('base_ef3__parents') }}
+    where not is_deleted
+),
+keyed as (
+    select 
+        {{ dbt_utils.surrogate_key(
+            [
+                'tenant_code',
+                'lower(parent_unique_id)'
+            ]
+        ) }} as k_parent,
+        base_parents.*
+        {{ extract_extension(model_name=this.name, flatten=True) }}
+    from base_parents
+),
+deduped as (
+    {{
+        dbt_utils.deduplicate(
+            relation='keyed',
+            partition_by='k_parent', 
+            order_by='pull_timestamp desc'
```

Comment:
`order_by='api_year desc, pull_timestamp desc'`
When we collapse across years like this, we have to be careful about pull_timestamp: we could have refreshed a historic year more recently than the current year.